runelite-client/src/main/java/net/runelite/client/plugins/inventoryviewer/InventoryViewerOverlay.java

```diff
@@ -75,7 +76,8 @@ public Dimension render(Graphics2D graphics)
 			return null;
 		}
 
-		if (client.getVar(VarClientInt.INVENTORY_TAB) == 3 && config.hideIfInventoryActive())
+		if ((client.getVar(VarClientInt.INVENTORY_TAB) == 3 || client.getWidget(WidgetInfo.BANK_CONTAINER) != null)
```

Comment:
Does this work both in fixed and resizable viewports after you close the bank? The bank widget, in some cases, is hidden rather than nulled after being closed--whether temporarily (until the next game tick) or for longer. This should ensure that is checked.

Reply:
I've checked both resizeable, full screen, and fixed and this fix works for all of them